src/snyk/common/languageServer/settings.ts

```diff
@@ -24,6 +24,7 @@ export type ServerSettings = {
   // Reporting and telemetry
   sendErrorReports?: string;
   enableTelemetry?: string;
+  enableAnalytics?: boolean;
```

Comment:
I'm not really sure why the other flags are handled as `string`. I decided to handle `enableAnalytics` as `boolean` but more than happy to change to `string` if that's the correct way :)

Reply:
A historical flaw/bug - using boolean here is correct.